clickhouse_options.go

```diff
@@ -49,12 +49,12 @@ const (
 	ConnOpenRoundRobin
 )
 
-type InterfaceType int
+type Protocol int
 
 const (
-	NativeInterface InterfaceType = iota
-	HttpInterface
-	HttpsInterface
+	Native Protocol = iota
+	Http
```

Comment:
Please use `HTTP` and `HTTPS` here as per https://github.com/golang/go/wiki/CodeReviewComments#initialisms